project/Dependencies.scala

```diff
@@ -6,21 +6,32 @@ import scala.collection.Seq
 import sbt._
 
 object Dependencies {
+
+  private val azureVersion = "10.1.0"
+  private val rxjavaVersion = "2.2.2"
   private val catsEffectVersion = "1.0.0"
   private val fs2Version = "1.0.0"
+  private val nettyVersion = "4.1.28.Final"
   private val quasarVersion = IO.read(file("./quasar-version")).trim
   private val qdataVersion = IO.read(file("./qdata-version")).trim
   private val shimsVersion = "1.2.1"
+  private val slf4jVersion = "1.7.25"
   private val specsVersion = "4.1.2"
 
   // direct as well as transitive deps need to be in sync with quasar's deps
   def datasourceCore = Seq(
-    "com.slamdata"           %% "qdata-json"          % qdataVersion,
     "com.codecommit"         %% "shims"               % shimsVersion,
+    "com.microsoft.azure"    %  "azure-storage-blob"  % azureVersion,
+    "com.slamdata"           %% "qdata-json"          % qdataVersion,
+    // this isn't strictly necessary but takes advantage of native libs
+    // keep in sync with the version azure uses
+    "io.netty"               %  "netty-all"           % nettyVersion,
+    "io.reactivex.rxjava2"   %  "rxjava"              % rxjavaVersion,
```

Comment:
Wait, what's the purpose of these dependencies exactly? I'm not sure I understand the comment.

Reply:
Oh, I'll make it more explicit. This is only referring to the `netty` dep, not the `rxjava` one.

Just noticed from the debug logging that with just the netty deps provided by Azure, the native jars like `netty-transport-native-kqueue`, `netty-transport-native-unix-common` and `netty-transport-native-epoll` are not there, so can't be taken advantage of.

Reply:
Ah fascinating and concerning. I approve.